tools/makeflash.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/sh
 # Usage:
 #
-#      ./makeflash.sh [-C size] <input dir> <output.img> [partitions]
+#      ./makeflash.sh <raw.img> [-C size] <input dir> <output.img> [partitions]
```

Comment:
Why the name of this arg is "raw.img"?  You are referring to the package name, raw.img it's a bit confusing....